Resources/Prototypes/Body/Organs/vox.yml

```diff
@@ -1,3 +1,32 @@
+- type: entity # starlight start
+  parent: OrganHumanBrain
+  id: OrganVoxBrain
+  name: vox cortical stack
+  description: "A cortical stack from a vox, time to find a new sleeve."
+  components:
+  - type: BorgBrain
+  - type: Sprite
+    sprite: Mobs/Species/Vox/organs.rsi
+    layers:
+      - state: brain
+      - state: brain-unshaded
+        shader: unshaded
+  - type: SolutionContainerManager
+    solutions:
+      organ:
+        maxVol: 10
+        reagents:
+        - ReagentId: Silicon
+          Quantity: 10
+      food:
+        maxVol: 20
+        reagents:
+        - ReagentId: Licoxide
```

Comment:
I'm a little worried about grinding vox brains being a good way to get Licoxide as a traitor... generally I don't think it'll be too much of an issue, though, with the typical RR restrictions.